package.json

```diff
@@ -1,10 +1,12 @@
 {
-  "name": "Restaurant-page",
+  "name": "restaurant-page",
   "version": "1.0.0",
   "description": "Practice with DOM, dynamically providing a simple restaurant homepage. Only JavaScript is used to generate all the content of the website.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
```

Comment:
- [ ] Please include other necessary scripts. It should essentially resemble this.
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```